src/SAML2/AuthnRequest.php

```diff
@@ -101,6 +101,20 @@ class SAML2_AuthnRequest extends SAML2_Request
      */
     private $requestedAuthnContext;
 
+    /**
+     * @var SAML2_XML_saml_SubjectConfirmation[]
+     */
+    private $SubjectConfirmation = array();
```

Comment:
Feel free to ignore (or fix!) our inconsistent capitalization strategy next time and follow camelCase.